releases/release-1.22/release-notes/maps/pr-101496-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 101496
+releasenote:
+  text: Pod Affinity `NamespaceSelector` and the associated `CrossNamespaceAffinity`
```

Comment:
`podAffinity` is a key from the spec, may it should be one word and code blocked. Also when we have a feature or something like it, that moves to beta we try to phrase it as:
**FOO FAR BAR feature graduates to Beta and enabled by default.**
Can you try to rephrase it to fit the above?

Reply:
> `podAffinity` is a key from the spec, may it should be one word and code blocked. Also when we have a feature or something like it, that moves to beta we try to phrase it as:
> **FOO FAR BAR feature graduates to Beta and enabled by default.**
> Can you try to rephrase it to fit the above?

Done.